virtcontainers/kata_agent.go

```diff
@@ -142,6 +144,17 @@ var kataHostSharedDir = func() string {
 	return defaultKataHostSharedDir
 }
 
+func getPagesizeFromOpt(fsOpts []string) string {
+	//example options array: "rw", "relatime", "seclabel", "pagesize=2M"
+
+	for _, opt := range fsOpts {
+		if strings.Contains(opt, "pagesize=") {
```

Comment:
This should be `strings.HasPrefix()` to avoid matching things like "`foo_bar_pagesize=`" (and for symmetry with the `TrimPrefix()` call).